to_double was not declared in this scope: Triangulation_2.h

## Issue Details

Hi, I was trying to compile [openFOAM 4.x](https://github.com/OpenFOAM/OpenFOAM-4.x) as described by [their guide](https://openfoam.org/download/source/compiling-openfoam/), using the given parallel parameter _-j_ but get an error about _to_double_ in _Triangulation_2.h_, from _cgal_:

`/usr/include/CGAL/Triangulation_2.h:3012:30: error: ‘to_double’ was not declared in this scope, and no declarations were found by argument-dependent lookup at the point of instantiation [-fpermissive]
   const double px = to_double(get_approx(p).x());` (for a complete description: [cgal_error.txt](https://github.com/CGAL/cgal/files/1079900/cgal_error.txt))

## Environment

* Operating system Arch Linux x64
* Compiler: gcc (GCC) 7.1.1 20170528
* Release or debug mode: Release
* CGAL version: [4.10-1](https://www.archlinux.org/packages/community/x86_64/cgal/)
* Boost version: [1.64.0-1](https://www.archlinux.org/packages/extra/x86_64/boost-libs/)

--- a/Triangulation_2/include/CGAL/Triangulation_2.h
+++ b/Triangulation_2/include/CGAL/Triangulation_2.h
@@ -40,6 +40,7 @@
 #include <CGAL/Triangulation_face_base_2.h>
 #include <CGAL/Triangulation_line_face_circulator_2.h>
 #include <CGAL/spatial_sort.h>
+#include <CGAL/double.h>
 
 #include <boost/random/linear_congruential.hpp>
 #include <boost/random/uniform_smallint.hpp>

